shipyard-templates/shipyard_templates/messaging.py

```diff
@@ -12,3 +15,64 @@ def connect(self):
     EXIT_CODE_BAD_REQUEST = 203
     EXIT_CODE_FILE_NOT_FOUND = 204
     EXIT_CODE_UNKNOWN_ERROR = 249
+
+    @classmethod
+    def message_content_file_injection(self, message: str) -> str:
```

Comment:
I'm almost of the opinion that this should be a static method, not a class method, thoughts?

Reply:
I had the seem thought initially until i wanted to raise the exit code.